apps/taxes/src/modules/avatax/configuration/avatax-edit-address-validation.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Client } from "urql";
+import { AvataxClient } from "../avatax-client";
+import { AvataxConfig } from "../avatax-connection-schema";
+import { AvataxAddressValidationService } from "./avatax-address-validation.service";
+import { AvataxPatchInputTransformer } from "./avatax-patch-input-transformer";
+
+export class AvataxEditAddressValidationService {
+  constructor(private client: Client, private appId: string, private saleorApiUrl: string) {}
```

Comment:
please rewrtie to object destructuring, its easy co confuse string argument order